webpack/webpack.common.js

```diff
@@ -2,11 +2,11 @@ const path = require("path");
 const webpack = require("webpack");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
-const CleanWebpackPlugin = require("clean-webpack-plugin");
-const GitRevisionPlugin = require("git-revision-webpack-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const { GitRevisionPlugin } = require("git-revision-webpack-plugin");
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 
-const gitRevisionPlugin = new GitRevisionPlugin({ branch: true }); // 'rev-parse HEAD' is default command to find latest commit
+// const gitRevisionPlugin = new GitRevisionPlugin({ branch: true }); // 'rev-parse HEAD' is default command to find latest commit
```

Comment:
Is the intention to comment this back in at some point (needing a TODO)? Or just remove it?